db/seeds/site_settings.yml

```diff
@@ -313,3 +313,17 @@
   category: AdvancedSettings
   description: >
     The minimum score a question must have to qualify for selection for the Hot Posts sidebar and for being selected as random advertisement.
+
+- name: TableOfContentsThreshold
+  value: 5
+  value_type: integer
+  category: Display
+  description: >
+    The number of child posts before the TOC is shown. If the threashold is not met, do not show TOC at all.
+
+- name: TableOfContentsExcerptLength
+  value: 120
+  value_type: integer
+  category: Display
+  description: >
+    The length of the answer shown in the TOC, don't keep it too large, it may fill up the TOC and become unreadable.
```

Comment:
Same here, but I'd start a new sentence after the first "TOC".

Reply:
The number of characters to show in the answer excerpt in the TOC.  The TOC strips most formatting, so beware of large values -- the TOC might become hard to read.

--

Nitpick: the earlier message refers to "child posts"; this one refers to "answers".  Please pick one and use it in both places.